setup.cfg

```diff
@@ -29,9 +29,6 @@ package_dir =
     rocksdb = rocksdb
 include_package_data = True
 zip_safe = False
-setup_requires =
-    cython >= 0.20
-    setuptools >= 25
```

Comment:
https://setuptools.pypa.io/en/stable/userguide/dependency_management.html#package-requirement According to this `setup_requires` is considered deprecated. I deleted this because I was getting warnings. I can revert it if you prefer.

Reply:
If it is causing warnings already, let's leave it out.